src/main/java/com/google/api/codegen/transformer/StaticLangApiMethodTransformer.java

```diff
@@ -87,7 +88,7 @@ public StaticLangApiMethodView generatePagedFlattenedMethod(
         additionalParams,
         Synchronicity.Sync,
         methodViewBuilder,
-        Arrays.asList(ClientMethodType.PagedFlattenedMethod));
+        Arrays.asList(CallingForm.FlattenedPaging));
```

Comment:
FYI, in general `Collections.singletonList` is more efficient, but no need to do it in this PR.

Reply:
Ack.